radio/src/gui/Taranis/menu_general_sdmanager.cpp

```diff
@@ -36,7 +36,7 @@
 
 #include "../../opentx.h"
 
-#define REFRESH_FILES() reusableBuffer.sdmanager.offset = 65535
+#define REFRESH_FILES() reusableBuffer.sdmanager.offset = 65535; m_posVert = 0;
```

Comment:
This is dangerous, you should surround these 2 functions by a do { ... } while(0)